### ISSUE TITLE
Consider using sccache

### ISSUE THREAD

**Issue by pornel** [imazen/imageflow#171]
https://github.com/mozilla/sccache

```sh
export RUSTC_WRAPPER=$(which sccache)
```

it can store compiled products in S3, so may be a nice alternative to caching build dirs in travis.

**Comment by nathanaeljones** [comment]
Yeah, I should do a comparison test at some point. Caching is currently really fast, though, and I use Incremental mode for everything except tagged releases.
